source/common/common/key_value_store_base.cc

```diff
@@ -31,6 +31,14 @@ absl::optional<absl::string_view> getToken(absl::string_view& contents, std::str
   return token;
 }
 
+bool checkForTtl(absl::string_view& contents) {
+  if (contents.size() > 3 && contents.substr(0, 3) == "TTL") {
```

Comment:
can we factor out into a variable and then use TTL_KEY.size() instead of hard coding 3?

Reply:
This is a great point, I'm probably supposed to put "TTL" somewhere as a constant and just reference it from there.

Tricky though... I need to access "TTL" in both checkForTtl and FileBasedKeyValueStore::flush.  I think I'd have to refactor these files to share a constant. lol

Reply:
True, I think the best option is to put it in key_value_store_base.h, since FileBasedKeyValueStore needs to depend on that file already.